Makefile

```diff
@@ -41,7 +41,7 @@ integ-test:
 	@echo "Building ecs-cli..."
 	./scripts/build_binary.sh ./bin/local
 	@echo "Running integration tests..."
-	go test -tags integ -v ./ecs-cli/integ/e2e/...
+	go test -timeout 60m -tags integ -v ./ecs-cli/integ/e2e/...
```

Comment:
just curious: do we know how long these tests are taking to run on average? wondering if we can track length of test runs in CloudWatch so we can keep an eye on things as we add more tests.

Reply:
After making the tests run in parallel, it came down to ~11 mins from ~17mins.

The default `go test` timeout is 10m, that's why i had to override that value. I picked 60m to be consistent with the codebuild timeout.